Makefile

```diff
@@ -28,3 +28,88 @@ binary/clean: ## clean ipxe binaries, upstream ipxe source code directory, and i
 		rm -rf binary/ipxe.efi binary/snp.efi binary/undionly.kpxe
 		rm -rf upstream-*
 		rm -rf ipxe-*
+
+.PHONY: test
+test: ## run unit tests
+	go test -v -covermode=count ./...
+
+.PHONY: cover
+cover: ## Run unit tests with coverage report
+	go test -coverprofile=coverage.out ./... || true
+	go tool cover -func=coverage.out
+
+# BEGIN: lint-install /Users/jacobweinstock/repos/tinkerbell/boots-ipxe
```

Comment:
this makefile was really clean and easy to read but now that lint-install threw in its pieces it feels like a lot. Point lint-install to manage `lint.mk` and then including it in Makefile is pretty nice. The human generated stuff stays easy to read/manage and the computer generated stuff is in its own little area (file).

Reply:
sounds good. will move the linting to lint.mk